src/stk/_internal/topology_graphs/cage/m6l8l12_cuboctahedron.py

```diff
@@ -0,0 +1,221 @@
+"""
+M6L8L12 Cuboctahedron
+==========
+
+"""
```

Comment:
no need to add docstrings to internal modules

Reply:
I added the tests to both files above, however when I run the tests locally I get an `'CallSpec2' object has no attribute 'funcargs'` message... I guess it might be a problem of the pytest version I have installed? 
I am not sure, it is the first time I have to run a real test :rofl:

Reply:
Ahh yes, this issue should be solved in the latest version of `stk` by limiting the pytest version to < 8 (see the `pyproject.toml` file). You may need to reinstall your development version with `just dev` to get the tests working.